terraform/api_gateway.tf.json.template.py

```diff
@@ -649,6 +649,20 @@ def for_domain(cls, domain):
                     app.name: {
                         'name': '/aws/apigateway/' + config.qualified_resource_name(app.name),
                         'retention_in_days': config.audit_log_retention_days,
+                    },
+                    f'{app.name}_api_execution': {
+                        'name': 'API-Gateway-Execution-Logs_'
+                                '${aws_api_gateway_rest_api.%s.id}'
+                                '/%s' % (app.name, config.main_deployment_stage),
+                        'retention_in_days': config.audit_log_retention_days,
+                    },
+                    **{
+                        f'{resource_name}_lambda': {
+                            'name': f'/aws/lambda/{resource['function_name']}',
+                            'retention_in_days': config.audit_log_retention_days
+                        }
+                        for resource_name, resource
+                        in chalice.tf_config(app.name)['resource']['aws_lambda_function'].items()
```

Comment:
PL

Reply:
Note: moving these `aws_cloudwatch_log_group` to `tf_config()` required moving the call of `tf_config()` up so it wasn't in the same nested block as the other `aws_cloudwatch_log_group`.

Old:
```
{
    'resource': [
        *(
            {
                **chalice.tf_config(app.name)['resource'],
                'aws_cloudwatch_log_group' : ...
            }
            for app in apps
        )
    ]
}
```

New:
```
{
    'resource': [
        *(
            **chalice.tf_config(app.name)['resource'],
            for app in apps
        ),
        *(
            {
                'aws_cloudwatch_log_group' : ...
            }
            for app in apps
        )
    ]
}
```